lib/screen/detail/artwork_detail_page.dart

```diff
@@ -83,12 +84,25 @@ class _ArtworkDetailPageState extends State<ArtworkDetailPage>
     );
   }
 
+  void _manualShare(String caption, String url) async {
+    final twitterUrl = "${SocialApp.twitterPrefix}?url=$url&text=$caption";
+    final twitterUri = Uri.parse(twitterUrl);
+    if (await canLaunchUrl(twitterUri)) {
+      launchUrl(twitterUri, mode: LaunchMode.externalApplication);
+    }
+  }
+
   void _shareTwitter(AssetToken token) {
     final prefix = Environment.tokenWebviewPrefix;
     final url = '$prefix/token/${token.id}';
     final caption = widget.payload.twitterCaption;
-    final hashtags = ['digitalartwallet', 'NFT'];
-    SocialShare.shareTwitter(caption!, hashtags: hashtags, url: url);
+    SocialShare.checkInstalledAppsForShare().then((data) {
+      if (data?[SocialApp.twitter]) {
+        SocialShare.shareTwitter(caption!, url: url);
```

Comment:
maybe you can add `caption = widget.payload.twitterCaption ?? "";` to prevent force nullable var `caption!`